File: Android/app/src/main/java/StaffScreen/Order/Request/CreateOrderRequest.java
package StaffScreen.Order.Request;

import java.util.List;

public class CreateOrderRequest{
    private int tableId;
    private List<OrderItems> items;

    public CreateOrderRequest() {
    }

    public CreateOrderRequest(int tableId, List<OrderItems> items) {
        this.tableId = tableId;
        this.items = items;
    }

    public int getTableId() {
        return tableId;
    }

    public void setTableId(int tableId) {
        this.tableId = tableId;
    }

    public List<OrderItems> getItems() {
        return items;
    }

    public void setItems(List<OrderItems> items) {
        this.items = items;
    }
}

File: API/HE170242_HaiTD_WebAPI/WebAPI/Request/CreateOrderRequest.cs
namespace WebAPI.Request
{
    public class CreateOrderRequest
    {
        public int tableId { get; set; }
        public List<billlItems> items { get; set; }
    }
    public class billlItems
    {
        public int menuId { get; set; }
        public int quantity { get; set; }
        public double price { get; set; }
    }
}
